src/main/java/io/leonis/torch/handlers/MultiWindowListener.java

```diff
@@ -0,0 +1,31 @@
+package io.leonis.torch.handlers;
+
+import com.googlecode.lanterna.gui2.*;
+import com.googlecode.lanterna.gui2.TextGUI.Listener;
+import com.googlecode.lanterna.input.KeyStroke;
+
+/**
+ * The Interface MultiWindowListener
+ *
+ * Extends the {@link Listener} interface to work for {@link WindowBasedTextGUI}.
+ *
+ * @author Jeroen de Jong
+ */
+public interface MultiWindowListener extends Listener {
+
+  @Override
+  default boolean onUnhandledKeyStroke(final TextGUI textGUI, final KeyStroke keyStroke) {
+    return textGUI instanceof WindowBasedTextGUI && onUnhandledKeyStroke(
```

Comment:
i would break right before `&&` and keep  the call to `onUnhandledKeyStroke` with its arguments

Reply:
lel autoformatting; will fix.